mithril-common/Cargo.toml

```diff
@@ -47,7 +47,7 @@ kes-summed-ed25519 = { version = "0.2.1", features = [
     "serde_enabled",
     "sk_clone_enabled",
 ] }
-mithril-stm = { path = "../mithril-stm", version = "0.3", default-features = false }
+mithril-stm = { path = "../mithril-stm", version = "0.3", default-features = false, features = ["batch-verify-aggregates"] }
```

Comment:
Do we need to use the `batch-verify-aggregates` here as it is default now?

Reply:
Since we disable default-feature to allow backend switch: yes, we need to repeat the feature here since the code is still gated behind it.